test/e2e/client_wrappers.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
```

Comment:
Can you put this file into a e2e/util pkg?

Reply:
Not sure if I would recommend importing these, the interfaces are going to change a whole bunch based on my needs. They're not full wrappers, you can see some of the calls are missing some parameters and substitute default ones. If a test ends up needing to supply those parameters at some point I will just change the interface as needed